docs/pages/setup/operations/upgrading.mdx

```diff
@@ -38,11 +38,16 @@ compatible.
 
 When running multiple binaries of Teleport within a cluster, the following rules apply:
 
-- **Patch and minor** versions are always compatible, for example, any 5.0.1 component will work with any 5.0.3 component and 6.1.0 component will work with any 6.7.0 component.
-- Major versions are always compatible with the **previous** major release. This means you must not attempt to upgrade from 5.x.x straight to 7.x.x. You must upgrade to 6.x.x first.
-- The above applies to both clients and servers. For example, a 6.x.x Proxy Service is
-  compatible with 5.x.x Nodes and 5.x.x `tsh`. But we don't guarantee that a
-  7.x.x `tsh` will work with a 5.x.x Proxy Service.
+- **Patch and minor** versions are always compatible, for example, any 8.0.1
+  component will work with any 8.0.3 component and any 8.1.0 component will work
+  with any 8.3.0 component.
+- Major versions are always compatible with the **previous** major release. This
+  means you must not attempt to upgrade from 6.x.x straight to 8.x.x. You must
+  upgrade to 7.x.x first.
+  - Servers support clients that are 1 major version behind, but do not support
```

Comment:
What would you think of replacing the second bullet with this bullet? The third bullet is not much longer than the second bullet, but is more helpful and specific while providing the same general information.

Reply:
Sure!

Reply:
I combined them but kept the note about not upgrading from 6 to 8, as that's important.